test/lib/Elastica/Test/ScriptFileTest.php

```diff
@@ -53,7 +53,7 @@ public function testConstructor()
         $expected = array(
             'script_file' => $value,
         );
-        $this->assertEquals($value, $scriptFile->getScriptFile());
```

Comment:
As getScriptFile is not released yet I suggest we don't call this change a BC change. See https://github.com/ruflin/Elastica/blob/master/CHANGELOG.md Or should we create a proxy method getScriptFile in the scriptFile object?